test/runtests.jl

```diff
@@ -186,7 +188,8 @@ end
 # Issue #27 - Mixup in lifting of getfield
 let var"'" = bwd
     @test (x->x^5)''(1.0) == 20.
-    @test (x->x^5)'''(1.0) == 60.
+    # Higher order control flow not yet supported
+    @test_broken (x->x^5)'''(1.0) == 60.
```

Comment:
```suggestion
    @test (x->(x*x)*(x*x)*x)''' == 60.
    # Higher order control flow not yet supported (https://github.com/JuliaDiff/Diffractor.jl/issues/24)
    @test_broken (x->x^5)'''(1.0) == 60.
```

Reply:
Mind if I commit this and merge?

Reply:
Go for it

Reply:
It was green before! Now fails at a line outside suggestion box range.

Reply:
That line tests inference quality. Since CI is running nightly, that might have regressed. I think it's fine to merge anyway, and we'll go back and fix it.